installer/pkg/config-generator/tls.go

```diff
@@ -90,6 +90,15 @@ func (c *ConfigGenerator) GenerateTLSConfig(clusterDir string) error {
 		return fmt.Errorf("failed to generate etcd CA: %v", err)
 	}
 
+	err = copyFile(filepath.Join(clusterDir, etcdCAKeyPath), filepath.Join(clusterDir, "generated/tls/etcd-client-ca.key"))
+	if err != nil {
+		return fmt.Errorf("failed to import kube CA cert into ingress-ca.crt: %v", err)
+	}
+	err = copyFile(filepath.Join(clusterDir, etcdCACertPath), filepath.Join(clusterDir, "generated/tls/etcd-client-ca.crt"))
```

Comment:
This new line, and the `copyFile` above, will conflict with #38.  I don't mind rebasing #38 if this lands first (it would be a pretty easy rebase), but thought I'd give a heads up.